src/Security/Security/Models/SqlVulnerabilityAssessment/PSSqlVulnerabilityAssessmentRemediation.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.SecurityCenter.Models.SqlVulnerabilityAssessment
+{
+    public class PSSqlVulnerabilityAssessmentRemediation
+    {
+        public string Description { get; set; }
+        public string[] Scripts { get; set; }
+        public bool Automated { get; set; }
+        public string PortalLink { get; set; }
+
+        public override string ToString()
+        {
+            return $"{{\r\n" +
+                $"{Description}\r\n" +
+                $"IsAutimated: {Automated}\r\n" +
+                $"Portal Link: {PortalLink}\r\n" +
+                $"Script:\r\n" +
+                (Scripts.Any() ? $"{string.Join("\r\n", Scripts)}\r\n" : "{}\r\n") +
```

Comment:
Please replace \r\n with Environment.NewLine to be compatible in different platforms.

Reply:
pending